app/messaging/update-requests-awaiting-debt-data.js

```diff
@@ -3,13 +3,17 @@ const { getHoldCategoryId } = require('../holds')
 
 const updateRequestsAwaitingDebtData = async (paymentRequest) => {
   if (!paymentRequest.debtType) {
-    throw new Error(`Payment request does not include debt data: ${paymentRequest.invoiceNumber}`)
+    const error = new Error(`Payment request does not include debt data: ${paymentRequest.invoiceNumber}`)
```

Comment:
does this need cause, just for consistency?

Reply:
cause is for building up a stack trace of errors so if one error causes another you can chain them. This is the first error thrown in this chain so there is no cause to attach.